### PR TITLE
SK-1748-TRANS-WEB-Emails-Zimbra-BETA

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/faq-zimbra/guide.pt-pt.md [hunk 1]
```diff
@@ -24,6 +24,10 @@ Zimbra propõe numerosas funcionalidades que serão brevemente propostas nos cat
 
 Zimbra oferece as mesmas funcionalidades e uma ergonomia semelhante ao OWA. Um guia de utilização do Zimbra está desde já disponível em [este endereço](/pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_zimbra).
 
+### Como posso configurar o meu endereço de e-mail num software de e-mail?
+
+Consulte a nossa página « [Configurar o seu endereço de e-mail Zimbra num programa de correio eletrónico](/pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_apps) ».
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/faq-zimbra/guide.pt-pt.md:29]:
```suggestion
Consulte a nossa página "[Configurar o seu endereço de e-mail Zimbra num programa de correio eletrónico](/pages/web_cloud/email_and_collaborative_solutions/zimbra/mail_apps)".
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/faq-zimbra/guide.de-de.md [hunk 2]
```diff
@@ -24,6 +24,10 @@ Zimbra bietet zahlreiche Funktionen, die in Kürze in den Katalogen von OVHcloud
 
 Zimbra bietet vergleichbare Funktionen und ein ähnliches Nutzererlebnis wie OWA. Eine Anleitung zur Verwendung von Zimbra finden Sie bereits unter [dieser Adresse](/pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_zimbra).
 
+### Wie konfiguriere ich meine E-Mail-Adresse auf einem E-Mail-Client?
+
+Besuchen Sie unsere Seite „[Zimbra-E-Mail-Adresse auf einem E-Mail-Client konfigurieren](/pages/web_cloud/email_and_collaborative_solutions/zimbra/zimbra_mail_apps)“.
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/faq-zimbra/guide.de-de.md:29]:
```suggestion
Besuchen Sie unsere Seite "[Zimbra-E-Mail-Adresse auf einem E-Mail-Client konfigurieren](/pages/web_cloud/email_and_collaborative_solutions/zimbra/zimbra_mail_apps)".
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/faq-zimbra/guide.en-gb.md [hunk 3]
```diff
@@ -20,6 +20,10 @@ Zimbra offers many features that will soon be available in OVHcloud catalogues.
 
 Zimbra offers similar features and a user experience close to OWA. A user guide for Zimbra is now available at [this address](/pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_zimbra).
 
+### How do I configure my email address on an email client?
+
+Visit our page “[Configuring your Zimbra email address in an email client](/pages/web_cloud/email_and_collaborative_solutions/zimbra/zimbra_mail_apps)”.
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/faq-zimbra/guide.en-gb.md:25]:
```suggestion
Visit our page "[Configuring your Zimbra email address in an email client](/pages/web_cloud/email_and_collaborative_solutions/zimbra/zimbra_mail_apps)".
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/faq-zimbra/guide.en-ie.md [hunk 4]
```diff
@@ -20,6 +20,10 @@ Zimbra offers many features that will soon be available in OVHcloud catalogues.
 
 Zimbra offers similar features and a user experience close to OWA. A user guide for Zimbra is now available at [this address](/pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_zimbra).
 
+### How do I configure my email address on an email client?
+
+Visit our page “[Configuring your Zimbra email address in an email client](/pages/web_cloud/email_and_collaborative_solutions/zimbra/zimbra_mail_apps)”.
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/faq-zimbra/guide.en-ie.md:25]:
```suggestion
Visit our page "[Configuring your Zimbra email address in an email client](/pages/web_cloud/email_and_collaborative_solutions/zimbra/zimbra_mail_apps)".
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/faq-zimbra/guide.es-es.md [hunk 5]
```diff
@@ -24,6 +24,10 @@ Zimbra ofrece numerosas funcionalidades que pronto estarán disponibles en los c
 
 Zimbra ofrece las mismas funcionalidades y una usabilidad cercana a OWA. Ya está disponible una guía del usuario de Zimbra en [esta dirección](/pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_zimbra).
 
+### ¿Cómo puedo configurar mi dirección de correo electrónico en un cliente de correo?
+
+Consulte nuestra página «[Configurar una dirección de correo electrónico de Zimbra en un cliente de correo](/pages/web_cloud/email_and_collaborative_solutions/zimbra/zimbra_mail_apps) ».
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/faq-zimbra/guide.es-es.md:29]:
```suggestion
Consulte nuestra página "[Configurar una dirección de correo electrónico de Zimbra en un cliente de correo](/pages/web_cloud/email_and_collaborative_solutions/zimbra/zimbra_mail_apps)".
```

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/getting_started_zimbra/guide.en-gb.md [hunk 6]
```diff
@@ -42,10 +45,14 @@ To access your Zimbra service, log in to your [OVHcloud Control Panel](/links/ma
...
 Before you start configuring your Zimbra email accounts, take note of the three elements that structure your Zimbra service hierarchically:
 
-- [**Organization**](#organizations): It enables domain names to be grouped together in order to associate them. **You don't need to create an organization to create an email account**.
+- [**Organization**](#organizations): It enables domain names to be grouped together in order to associate them.kkk
 - [**Domain name**](#domains): It is essential to create an email account. You will need to manage at least one domain name via the OVHcloud Control Panel, and add it to your Zimbra service.
 - [**Email accounts**](#emails): By using the domain names added to your Zimbra service, you can create an email address.
 
+> [!primary]
+>
+> *Organization* is used to represent an entity (a company, association, personal project, etc.). It enables email accounts to be partitioned, specific security policies to be applied (future feature) and rights to be delegated to an organization (future feature). By using organizations, you can make it easier to browse and manage your Zimbra platform.
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/getting_started_zimbra/guide.en-gb.md]:
```suggestion
> An *organization* is used to represent an entity (a company, association, personal project, etc.). It enables email accounts to be partitioned, specific security policies to be applied (future feature) and rights to be delegated to an organization (future feature). By using organizations, you can make it easier to browse and manage your Zimbra platform.
```

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/getting_started_zimbra/guide.es-es.md [hunk 7]
```diff
@@ -74,18 +85,26 @@ Para retirar el filtro, simplemente haga clic en la cruz del filtro.
 
 ### Dominios <a name="domains"></a>
 
+> [!warning]
+>
+> Para un funcionamiento óptimo cuando utilice el mismo nombre de dominio entre los productos OVHcloud [Exchange](/links/web/emails-hosted-exchange), [E-mail Pro](/links/web/email-pro) y Zimbra, es necesario configurar el dominio en « no autoritativo ». Para más información sobre cómo configurar un dominio sin autorización en una plataforma Exchange o Email Pro, consulte nuestra guía [Añadir un dominio a una plataforma de correo](/pages/web_cloud/email_and_colaborative_solutions/microsoft_exchange/exchange_adding_domain).
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/getting_started_zimbra/guide.es-es.md]:
```suggestion
> Para un funcionamiento óptimo cuando utilice el mismo nombre de dominio entre los productos OVHcloud [Exchange](/links/web/emails-hosted-exchange), [E-mail Pro](/links/web/email-pro) y Zimbra, es necesario configurar el dominio en "no autoritativo". Para más información sobre cómo configurar un dominio sin autorización en una plataforma Exchange o Email Pro, consulte nuestra guía [Añadir un dominio a una plataforma de correo](/pages/web_cloud/email_and_colaborative_solutions/microsoft_exchange/exchange_adding_domain).
```

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/getting_started_zimbra/guide.es-es.md [hunk 8]
```diff
@@ -116,21 +137,34 @@ Complete la información que se muestra.
...
+- **Nombre a mostrar**  : Introduzca el nombre que quiera que figure como remitente cuando envíe mensajes desde esta dirección.
 - **Contraseña** : Establezca una contraseña segura que incluya (como mínimo) 9 caracteres, una mayúscula, una minúscula y una cifra. Por motivos de seguridad, no utilice la misma contraseña dos veces. Elija un nombre que no guarde relación con sus datos personales (por ejemplo, no incluya su nombre, apellidos ni fecha de nacimiento). Cámbielo regularmente.
 
 > [!warning]
 >
 > La elección de la contraseña debe respetar las siguientes condiciones :
 >
-> - Mínimo 9 caracteres
-> - Máximo 30 caracteres
+>: mínimo 10 caracteres
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/getting_started_zimbra/guide.es-es.md]:
```suggestion
> - Mínimo 10 caracteres
```

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/getting_started_zimbra/guide.es-es.md [hunk 9]
```diff
@@ -116,21 +137,34 @@ Complete la información que se muestra.
...
 
 Haga clic en `Confirmar`{.action} para crear la cuenta.
 
+![zimbra](images/zimbra_emailaccounts_add.png){.thumbnail .w-400}
+
+### Consultar su cuenta de correo <a name="mail-consult"></a>
+
+Para consultar su cuenta de correo:
+
+- Conéctese al [webmail](/links/web/email) desde un navegador de internet e introduzca su dirección de correo y contraseña. Para más información, consulte nuestra página «[Utilizar el webmail Zimbra](/pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_zimbra) ».
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/getting_started_zimbra/guide.es-es.md]:
```suggestion
- Conéctese al [webmail](/links/web/email) desde un navegador de internet e introduzca su dirección de correo y contraseña. Para más información, consulte nuestra página "[Utilizar el webmail Zimbra](/pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_zimbra) ".
```
Please search and replace all « » with " on all DE, EN, ES, IT, PL, PT files.

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/zimbra_mail_apps/guide.it-it.md [hunk 10]
```diff
@@ -0,0 +1,118 @@
+---
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/zimbra_mail_apps/guide.it-it.md]: something is wrong with this header (GitHub coloration is different)
Check extra spaces

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/zimbra_mail_apps/guide.pl-pl.md [hunk 11]
```diff
@@ -0,0 +1,118 @@
+---
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/zimbra_mail_apps/guide.pl-pl.md]: Header to check

File: pages/web_cloud/email_and_collaborative_solutions/zimbra/zimbra_mail_apps/guide.pt-pt.md [hunk 12]
```diff
@@ -0,0 +1,118 @@
+---
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/zimbra/zimbra_mail_apps/guide.pt-pt.md]: Header to check

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**